src/blog/2018-12-25/stryker4s-0-1-0-released.pug

```diff
@@ -0,0 +1,33 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        It's taken a little longer than we maybe would've liked, but we are very proud of this first release. In this blog post we'll tell you about why running with the sbt plugin is a big improvement, and some of the features you can use today for Stryker4s. Although it's still possible to use the command-runner for non-sbt projects, we decided a sbt plugin was an important for the first release.
```

Comment:
This is fine, but personally I wouldn't start out with a kind of negative message like this.